Readme.md

```diff
@@ -58,7 +58,7 @@ Available options:
   * Default: `Routes`
 * `prefix` - String representing a url path to prepend to all paths.
   * Example: `http://yourdomain.com`. This will cause route helpers to generate full path only.
-  * Default: blank
+  * Default: value of Rails.application.config.relative_url_root or RAILS_RELATIVE_URL_ROOT env variable
```

Comment:
Default: `Rails.application.config.relative_url_root`. I think this is enough. People that use ENV aware of that.